patientsearch/src/js/components/Utility.js

```diff
@@ -1,9 +1,14 @@
-export function sendRequest (url) {
+export function sendRequest (url, params) {
+    params = params || {};
     // Return a new promise.
     return new Promise(function(resolve, reject) {
       // Do the usual XHR stuff
       var req = new XMLHttpRequest();
       req.open('GET', url);
+      if (params.nocache) {
+        // via Cache-Control header:
+        req.setRequestHeader("Cache-Control", "no-cache, no-store, max-age=0");
```

Comment:
I'm guessing @ivan-c will suggest the same removal of `no-store` and `max-age` :)